### ISSUE TITLE
Run ELF on a computer without a DVD drive

### ISSUE THREAD

**Issue by jonnygrant** [PCSX2/pcsx2#3896]
***PCSX2 version:***
``(For example, stable build v1.6.0 (or) development builds v1.7.0-dev-xxx)``

1.6.0
- **Notes**:
  - It's preferred to only report bugs which are present on the [latest development builds](https://pcsx2.net/download/development.html).
  - The PSX backward compatibility mode is still in its early stage of development. Issue reports are too early at this point.

***PCSX2 options:***
``(For example, Clamping/Rounding, Speedhacks, GameFixes)``

- **Note**: Generally it's recommended to have settings as default as possible and use the ``Safest`` preset. If there's no issue with these settings but there is an issue with some other non-default option, then please state it explicitly.

***Plugins used:***
``(Relevant/Non-default settings of GS/PAD/SPU2/CDVD/USB plugins)``

- **Note**: In case the issue is a graphical problem with the GSdx plugin, please provide basic information like ``Renderer/Internal resolution/CRC Hack level/Other relevant options`` which were used to reproduce the issue. Additionally, providing a [GS dump](https://forums.pcsx2.net/Thread-How-to-create-a-proper-GS-dump) of the issue would also be helpful.

***Description of the issue:***
``(What's the problem? - Screenshots showing the issue if applicable)``

Is it possible to run an ELF without a DVD drive?
the log has fails  CDVDios Error: No filename specified.

failure.  Plugins unloaded.
HLE Host: Set 'host:' root path to: C:\Users\jg\Documents\PCSX2\

Initializing plugins...
	Init GS
	Init PAD
	Init SPU2
	Init CDVD
	Init USB
	Init FW
	Init DEV9
Plugins initialized successfully.

Patches: No CRC found, using 00000000 instead.
Opening plugins...
	Opening GS
	Opening PAD
	Opening SPU2
	Opening CDVD
CDVDiso Error: No filename specified.
Closing plugins...
	Closing SPU2
	Closing PAD
	Closing GS
Plugins closed successfully.
Shutting down plugins...
Current Renderer: Direct3D 11 (Hardware renderer)
GSdx Lookup CRC:0
GSdx Lookup CRC:0
Plugins shutdown successfully.
(pxActionEvent) CDVD plugin failed to open!(thread:EE Core)
Bios Found: Japan   v01.00(17/01/2000)  Console
User-canceled plugin configuration after plugin initialization 
failure.  Plugins unloaded.

***How to reproduce the issue:***
``(Detailed step by step guide on how to replicate the issue)``

***Last known version to work:***
``(If the issue isn't present in an older PCSX2 version, it would be helpful to mention the full title of that version. Narrowing it down to the build/commit that first introduced the issue is even better!)``

***PC specifications:***
``(For example: CPU: Intel Core 2 Duo E4600 GPU: Gigabyte GTX 750Ti OS: Windows 10 64 Bit)``

**Comment by refractionpcsx2** [comment]
Pretty sure this has already been resolved in master, can you try the latest development build from here: https://buildbot.orphis.net/pcsx2/

**Comment by tadanokojin** [comment]
I think it is a duplicate of #3720. Test #3784.

**Comment by jonnygrant** [comment]
Many thanks for your replies Refraction and Taddanokojin

I got it working. In the end by setting "No disc"

Attached my ELF file. I compiled this back 24 Mar 2003

What I don't really recall is how I ever managed to test it, because I didn't have a ps2dev machine, naplink never worked for me. I believe I was disc swapping... that meant I needed to make sure everything worked first time I compiled it!

[balls.zip](https://github.com/PCSX2/pcsx2/files/5503708/balls.zip)
